source/views/sis/course-search/lib/format-day.js

```diff
@@ -20,3 +20,24 @@ export function formatDay(day: string) {
 			return 'Unknown Day'
 	}
 }
+
+export function formatDayAbbrev(day: string) {
+	switch (day) {
+		case 'Mo':
+			return 'M'
+		case 'Tu':
+			return 'T'
```

Comment:
I know SIS does 'MTWThF', but what do we think about doing 'MTuWThF'?

Reply:
I'd be ok with that I guess...haha I'll just have to get used to looking at TuTh instead of TTh

Reply:
I'm find with TTh – let's leave it.